portal-kernel/src/com/liferay/portal/kernel/util/PropsKeys.java

```diff
@@ -305,19 +305,6 @@
 		BLOGS_ENTRY_PREVIOUS_AND_NEXT_NAVIGATION_ENABLED =
 			"blogs.entry.previous.and.next.navigation.enabled";
 
-	/**
-	 * @deprecated As of Mueller (7.2.x), with no direct replacement
-	 */
-	@Deprecated
-	public static final String BLOGS_IMAGE_EXTENSIONS =
-		"blogs.image.extensions";
```

Comment:
@jeffwu0724 Please search "blogs.image.extensions" in VerifyProperties, you would see "blogs.image.extensions" was already added to _MODULARIZED_PORTAL_KEYS. In this case, you do not need to add it to _OBSOLETE_PORTAL_KEYS.

Please check all other properties you removed, if it is same case as "blogs.image.extensions", please remove your changes for it in VerifyProperties